backend-lucene/src/main/java/org/hibernate/search/v6poc/backend/lucene/types/dsl/impl/AbstractLuceneIndexSchemaFieldTypedContext.java

```diff
@@ -61,11 +60,6 @@ protected abstract void contribute(DeferredInitializationIndexFieldAccessor<T> r
 		return this;
 	}
 
-	@Override
-	public IndexSchemaFieldTypedContext<T> sortable(Sortable sortable) {
-		throw log.cannotUseSortableOnFieldType( relativeFieldName );
-	}
```

Comment:
I'm not sure it's totally useless. Especially if we want to remove this from the `GeoPoint` type.

Reply:
It was overridden in every subclass that actually exposed it, so it was useless and I was unable to test it. But we can add it back later, sure.